src/components/blog-post/BlogPostImage.js

```diff
@@ -1,16 +1,11 @@
 import React from 'react';
-import { GatsbyImage, getImage } from 'gatsby-plugin-image';
 
-const BlogPostImage = ({ headerImage, title }) => (
-    <div className="pt-12 bg-gradient-to-r from-primary to-secondary rounded-xl">
-        {headerImage && (
-            <GatsbyImage
-                alt={title}
-                image={getImage(headerImage[0].localFile)}
-                className="w-full rounded-xl"
-            />
-        )}
-    </div>
+const BlogPostImage = ({ contentImage, additionalClass }) => (
+    <img
+        src={contentImage}
+        alt={contentImage}
```

Comment:
```suggestion
const BlogPostImage = ({ contentImage, imageAlt = '', additionalClass = [] }) => (
    <img
        src={contentImage}
        alt={imageAlt}
```
Alt should never be a link to an image